pages/index.tsx

```diff
@@ -202,7 +178,7 @@ const Home: NextPage = ({ general, niche }: any) => {
                                 key={ 0 }
                                 data-tip='Instances open to furries of any kind with no specific topic'
 
-                                className={ `text-2xl tab tab-bordered h-fit w-full tooltip tooltip-primary w-full ${
```

Comment:
Both this line and the one 10 lines below it assign the "w-full" class twice, this has been fixed to only applying it once.